data/product.d/rhvh.conf

```diff
@@ -1,7 +1,17 @@
 # Anaconda configuration file for Red Hat Virtualization.
+#
+# RHVH is using image installation
+# kickstart presence + liveimg in it is requres for image installation
```

Comment:
Please fix the typo:

`# kickstart presence + liveimg in it is requ(i)res for image installation`

Although, this comment seems somehow grammatically incorrect even with the typo fix. Could you please rephrase it?

Reply:
"required" ?

Reply:
Fixed.